### PR TITLE
Add the 'action=install' label to crs

### PR DESCRIPTION
**Pull request by jakkab**
**Description**

Changes proposed in this pull request:

- remove cr labelling from the `installer.sh` script
- add default `action=install` label to cr templates
- refactor the `tiller-tls.sh` script

**Related issue(s)**
#3932

### CODE REVIEW

**Comment by jakkab** [comment]
pre-master-kyma-gke-upgrade-nosed

File: installation/scripts/installer.sh [hunk 1]
```diff
@@ -92,13 +91,13 @@ if [ ${LOCAL} ]; then
...
-
 echo -e "\nConfiguring sub-components"
 bash ${CURRENT_DIR}/configure-components.sh
 
-echo -e "\nGetting Helm secrets"
-bash ${CURRENT_DIR}/tiller-tls.sh
+echo -e "\nApplying combo yaml"
+kubectl apply -f - <<< "$COMBO_YAML"
+
+echo -e "\nDownloading Helm certificates"
```
Comment (Tomasz-Smelcerz-SAP) [inline installation/scripts/installer.sh]:
```suggestion
echo -e "\Setting up Helm certificates"
```

File: installation/scripts/installer.sh [hunk 2]
```diff
@@ -92,13 +91,13 @@ if [ ${LOCAL} ]; then
...
 fi
 
-kubectl apply -f - <<<"$COMBO_YAML"
-
 echo -e "\nConfiguring sub-components"
 bash ${CURRENT_DIR}/configure-components.sh
 
-echo -e "\nGetting Helm secrets"
-bash ${CURRENT_DIR}/tiller-tls.sh
+echo -e "\nApplying combo yaml"
```
Comment (Tomasz-Smelcerz-SAP) [inline installation/scripts/installer.sh]: I would put this section at the end (after Helm certificates), and log just: "Starting Kyma Installation" or something like that.

File: installation/scripts/installer.sh (hunk 1 above)
Comment (Demonsthere) [reply installation/scripts/installer.sh]: This is wrong, we are in fact downloading the certs from the kubernetes secrets to loca files, we do not set them :)

**Pull request merged**